.github/workflows/release.yml

```diff
@@ -21,7 +21,6 @@ env:
 jobs:
   changelog:
     name: Changelog PR or Release
-    if: ${{ github.repository_owner == 'withastro' }}
```

Comment:
Is there a reason this is moved into a step below? Shouldn't we skip this workflow entirely on forks?

Reply:
Didn't mean to change this, but yes, can totally skip the whole workflow.